neo/SmartContract/JsonSerializer.cs

```diff
@@ -26,7 +26,7 @@ public static JObject Serialize(StackItem item)
                     }
                 case ByteArray buffer:
                     {
-                        return buffer.GetString();
+                        return buffer.GetByteArray().ToHexString();
```

Comment:
If we do this
```Serialize(Deserialize("{"test":"abc"}"))```
We should obtain the same, and i think that is not ...

Reply:
I don't think there is an easy way to do this. Because strings and byte arrays are the same in NeoVM.

Reply:
I know... but maybe if the hex/base64 decode fail, can we use it as normal string?

Reply:
This rule is very strange.